### DISCUSSION TITLE
181

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#149] [General]
# 潮流周刊第181期 - 水培薄荷

封面图拍摄来源用于喝完的苏打水瓶子水培薄荷生根后的效果，直接用简单从盆里剪了几根薄荷插到水了就活了，太太太有生命力的感觉了，还长了很多根须。

https://weekly.tw93.fun/posts/181-%E6%B0%B4%E5%9F%B9%E8%96%84%E8%8D%B7

**Comment by i-vox** [comment]
有了GPT以后 大部分人都能很好的应用混乱的git指令了。

**Reply by tw93** [reply]
哈哈哈哈 也是，不过懂一点儿其实更好，知其所以然

**Comment by xcdmrCHP** [comment]
小建议：大标题`随便看看`应该可以改成`随便一下`，因为它包含了`随便看看`和`随便写写`

**Reply by tw93** [reply]
有道理  随便写写是后面我新加的一个的 可能是这周的一个生活总结

**Comment by JulyFinal** [comment]
原来薄荷也行! 我现在有个水培绿萝,接下来试试薄荷.

**Reply by tw93** [reply]
是的，还有红薯

**Comment by thetbw** [comment]
减餐半个月瘦了10斤，不过现在持续减餐也瘦不了了，还是要和锻炼配合，然后长期坚持

**Reply by tw93** [reply]
锻炼也比较重要了，我打算控制住肌肉的下降

**Reply by i-vox** [reply]
> 减餐半个月瘦了10斤，不过现在持续减餐也瘦不了了，还是要和锻炼配合，然后长期坚持

进入平台期了

**Comment by hesetiema** [comment]
其实很想知道肚子大了怎么减下去...体重降了就这个一直稳定的又大又圆...😒

**Reply by tw93** [reply]
需要配合锻炼

**Reply by VmzXv** [reply]
瘦的还不够，少吃多练继续减。

**Comment by TenTh0usand** [comment]
重庆小面也要戒😭

**Reply by tw93** [reply]
哈哈哈 慢慢来 等你瘦了 你就会很有信心

**Comment by loyaniu** [comment]
想问一下那个暗黑mac主题还能免费下载吗？没看到兑换码在哪里
